docs/faststore/docs/headless-cms/2-setting-up-the-headless-cms.mdx

```diff
@@ -177,11 +177,11 @@ CMS synced successfully...
 
 ### Step 3 - Checking your changes
 
-1. Access the VTEX Admin using the workspace you previously created (e.g., `https://{workspace}--{account}.myvtex.com/admin`).
+1. Access the VTEX Admin using the workspace you created earlier by navigating to the following path: `https://{workspace}--{account}.myvtex.com/admin`. Ensure to replace `{workspace}` with the designated workspace name and `{account}` with your VTEX account.
```

Comment:
Instead of "earlier", I think it would be better to refer to the specific section / step where the workspace was created.